public/js/lib/codemirror-mozilla.css

```diff
@@ -125,6 +125,14 @@ selector in floating-scrollbar-light.css across all platforms. */
   background-repeat: repeat-x;
 }
 
+.cm-selecting {
+  background: none;
+  border-width: 1px;
```

Comment:
this is a minor thing, but would you mind trying outline as well. I notice that border pushes the text, which is a slight visual tick that atom and i believe other editors avoid